.buildkite/pipeline_cpu_template.py

```diff
@@ -56,10 +56,18 @@ class BkStep(str, Enum):
         },
         "instances": ["m5d.metal", "m6i.metal", "m6a.metal"],
     },
+    "aarch64_cpu_templates": {
```

Comment:
`feat: addde aarch64 cpu tempalte buildkite group`: triple "d" in commit message

Reply:
fixed

Reply:
tempalte

Reply:
^ (commit message)